babel.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = api => {
+  api.cache(true)
+
+  const presets = [
+    [
+      '@babel/preset-env',
+      {
+        useBuiltIns: 'usage',
+        corejs: '3.0.0',
```

Comment:
[This](https://babeljs.io/docs/en/babel-preset-env#corejs) must be an integer or an object.